tox.ini

```diff
@@ -2,7 +2,7 @@
 # fakeroot -u tox --recreate
 
 [tox]
-envlist = py{34,35,36,37,38},flake8
+envlist = py{35,36,37,38,39},flake8
```

Comment:
don't drop py34 here, we still announce support for it in pypi package metadata.

(guess I will drop it **after** 1.1.14 release though)